### ISSUE TITLE
Error while enabling DiscordBridge, IllegalStateException ("settings.token") must not be null

### ISSUE THREAD

**Issue by Okramto** [the-obsidian/DiscordBridge#34]
```
[01:36:31 INFO]: [DiscordBridge] Enabling DiscordBridge v1.7.0
[01:36:31 ERROR]: Error occurred while enabling DiscordBridge v1.7.0 (Is it up to date?)
java.lang.IllegalStateException: plugin.config.getString("settings.token") must not be null
    at gg.obsidian.discordbridge.Configuration.load(Configuration.kt:43) ~[?:?]
    at gg.obsidian.discordbridge.Plugin.updateConfig(Plugin.kt:53) ~[?:?]
    at gg.obsidian.discordbridge.Plugin.onEnable(Plugin.kt:11) ~[?:?]
    at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:292) ~[spigot-1.9.2.jar:git-Spigot-5a40365-b70058a]
    at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340) [spigot-1.9.2.jar:git-Spigot-5a40365-b70058a]
    at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [spigot-1.9.2.jar:git-Spigot-5a40365-b70058a]
    at org.bukkit.craftbukkit.v1_9_R1.CraftServer.loadPlugin(CraftServer.java:361) [spigot-1.9.2.jar:git-Spigot-5a40365-b70058a]
    at org.bukkit.craftbukkit.v1_9_R1.CraftServer.enablePlugins(CraftServer.java:321) [spigot-1.9.2.jar:git-Spigot-5a40365-b70058a]
    at net.minecraft.server.v1_9_R1.MinecraftServer.t(MinecraftServer.java:411) [spigot-1.9.2.jar:git-Spigot-5a40365-b70058a]
    at net.minecraft.server.v1_9_R1.MinecraftServer.l(MinecraftServer.java:376) [spigot-1.9.2.jar:git-Spigot-5a40365-b70058a]
    at net.minecraft.server.v1_9_R1.MinecraftServer.a(MinecraftServer.java:331) [spigot-1.9.2.jar:git-Spigot-5a40365-b70058a]
    at net.minecraft.server.v1_9_R1.DedicatedServer.init(DedicatedServer.java:269) [spigot-1.9.2.jar:git-Spigot-5a40365-b70058a]
    at net.minecraft.server.v1_9_R1.MinecraftServer.run(MinecraftServer.java:527) [spigot-1.9.2.jar:git-Spigot-5a40365-b70058a]
    at java.lang.Thread.run(Thread.java:745) [?:1.8.0_77]
```

**Comment by ezenity** [comment]
I also receive the same error upon updating from v1.6.0 to v1.7.0

Running CraftBukkit version git-Spigot-a1b8bf3-f01861f (MC: 1.9.2) (Implementing API version 1.9.2-R0.1-SNAPSHOT)

DiscordBridge 1.7.0 Error (Image Form)
![DiscordBridge_1.7.0_Error](https://i.imgur.com/GihI5cI.png)

DiscordBridge 1.7.0 config.yml file, I removed my info however the way you see it is how i have it with my info inserted. 
![DiscordBridge_1.7.0_Config.yml](https://i.imgur.com/8qwBgM8.png)

EDIT:

I just downgraded to v1.6.0 to see if it worked. 

![DiscordBridge_1.6.0_Success](https://i.imgur.com/xG6DYXX.png)

**Comment by jacobwgillespie** [comment]
I think I fixed this in 1.7.1: https://github.com/the-obsidian/DiscordBridge/releases/tag/v1.7.1.

Let me know if that resolves it

**Comment by Okramto** [comment]
Issue resolved in 1.7.1
